APIGateway/APIGateway/configuration.json

```diff
@@ -0,0 +1,83 @@
+﻿{
+	"Routes": [
+		{
```

Comment:
Probably todo endpoint not expected to be here

Reply:
I put it to show that it actually works. I'll remove it from here